smac/epm/rf_with_instances.py

```diff
@@ -39,6 +39,7 @@ class RandomForestWithInstances(AbstractEPM):
 
     def __init__(self, types: np.ndarray,
                  bounds: typing.List[typing.Tuple[float, float]],
+                 seed: int = 42,
```

Comment:
GPs do not have a default seed. Why do RFs have one?

Reply:
I just removed the rf default seed.